File: Indie Python Programming Course/step07/7.01/7.1.08.py
'''

Создайте функцию count_letters, которая принимает на вход фразу и подсчитывает, какое количество в ней строчных(K) и заглавных (N) букв, 
все остальные символы игнорируются. Функция должна вывести на экран информацию о найденных буквах в определенном формате. 

Количество заглавных символов: N
Количество строчных символов: K

Вам необходимо написать только определение функции count_letters.

Sample Input 1:

Привет, Старина

Sample Output 1:

Количество заглавных символов: 2
Количество строчных символов: 11

Sample Input 2:

QWERTY

Sample Output 2:

Количество заглавных символов: 6
Количество строчных символов: 0

'''

def count_letters(word):
    cnt_upper = 0
    cnt_lower = 0
    for i in word:
        if i.isupper():
            cnt_upper += 1
        elif i.islower():
            cnt_lower += 1
    print(f'Количество заглавных символов: {cnt_upper}')
    print(f'Количество строчных символов: {cnt_lower}')

File: Fundamentals of C programming/step07/7.1/7.1.08.c
/*

Измените программу, написанную на прошлом шаге таким образом, чтобы каждое число выводилось столько раз, каково его значение. Например, число 55 должно выводиться 55 раз.

Входные данные:
Два натуральных числа A,B,A,B, таких, что B>A.B>A.

Выходные данные:
AA чисел AA , A+1A+1 чисел A+1A+1 и т.д. Каждое число занимает поле шириной в 4 символа.

Sample Input:

2 5

Sample Output:

2   2   3   3   3   4   4   4   4   5   5   5   5   5

*/

#include <stdio.h>

int main() {
    int a,b;
    scanf("%d %d",&a,&b);
    for (int c=a; c<b+1; c++){
        for (int d=0; d<=c-1; d++){
            printf("%4d",c);}
    }
    return 0;
}
